src/EditorFeatures/Core/Diagnostics/DiagnosticsClassificationTaggerProvider.cs

```diff
@@ -1,104 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This tagger is what faded out code.  But that's handled entirely by the LSP client now.